co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureMessageAmbassador.java

```diff
@@ -149,44 +138,120 @@ private synchronized void receiveInteraction(InfrastructureV2xMessageReception i
         log.info(infrastructureV2xMessageReception.getReceiverID() + " received V2X messages at time: "
                 + infrastructureV2xMessageReception.getTime() + ".");
         log.info("The received message is " + infrastructureV2xMessageReception.getMessage() + " .");
+    }
 
+    /**
+     * Creates an Ad-Hoc configuration object to represent the configuration of the
+     * Ad-Hoc interface used for communication between the infrastructure instance
+     * and other vehicles or components, and sends it to the RTI for exchange.
+     *
+     * @param reg the infrastructure registration message received from the RTI
+     */
+    private final void onDsrcRegistrationRequest(InfrastructureRegistrationMessage reg) {
+        // Create an InterfaceConfiguration object to represent the configuration of the
+        // Ad-Hoc interface
+        // Set the IP address and subnet mask to null for now
+        // Set the transmit power to 50 dBm and the maximum range to 100 meters
+        InterfaceConfiguration interfaceConfig = new InterfaceConfiguration.Builder(AdHocChannel.SCH1)
+                .ip(null)
+                .subnet(null)
+                .power(50)
```

Comment:
These are good for now, might be worth adding a TODO comment here for us to get back to making these parameters sometime, or writing up a JIRA ticket or something.